scripts/zones/Beadeaux/mobs/Old_Quadav.lua

```diff
@@ -12,7 +12,7 @@ entity.onMobDeath = function(mob, player, isKiller)
 end
 
 entity.onMobDespawn = function(mob)
-    xi.mob.phOnDespawn(mob, ID.mob.GE_DHA_EVILEYE_PH, 25, math.random(3600, 7200)) -- 1 to 2 hours
+    xi.mob.phOnDespawn(mob, ID.mob.GE_DHA_EVILEYE_PH, 25, 3600) -- 1 hour minimum
```

Comment:
spawn rate is likely a little high for its now less random cooldown, but it would be tough to pin down data either way. Maybe we just assume it's the same as the others till we know different? up to you, if you want to leave it as is I'm ok with that and will mark convo resolved.

Reply:
I've bumped down the spawn chance to 10% to match the other Quadav NMs in the zone.